### DISCUSSION TITLE
set pre-value

### DISCUSSION THREAD

**Discussion by 12xgde** [jomjol/AI-on-the-edge-device#1918] [Q&A]
in set pre-value sind die Werte raw <-> pre-value vertauscht seit v13.0.8 - bei Euch auch?

**Comment by caco3** [comment]
Bist Du sicher?
Kannst Du einen Screenshot zeigen (zusammen mit einem Screenshot der aktuellen Overview)?

Das Feld `Previous Value` enthält jeweils den aktuellen raw Wert, da man ja i.d.R. das setzen möchte.
